### DISCUSSION TITLE
Removing borders of adjacent ground markers

### DISCUSSION THREAD

**Discussion by PenguinMeat** [runelite/runelite#16219] [Ideas]
Simple idea that would purely improve the visual look of ground markers. This would not have any functional effect (and in some circumstances such as bossing could actually be harmful for player positioning). So having the option to "join" markers would be a nice cosmetic upgrade.

Example

Before:

![gimp-2 10_ltMjn8gqug](https://user-images.githubusercontent.com/6394808/214978287-43352dbf-f4d5-4bfe-a6c1-a90d40dac9cf.jpg)

After:

![gimp-2 10_9JBHBlExFC](https://user-images.githubusercontent.com/6394808/214978300-a690adcf-e0b5-455b-8e24-8fc27e58a635.jpg)

**Comment by Nightfirecat** [comment]
I'm not sure we should necessarily do this, however I've thought for a while that you should be able to mark areas (rectangles larger than 1 tile by 1 tile) in addition to tiles, which would resemble this for cases where a simple rectangle needed to be marked.
